detox/e2e/support/utils/email.ts

```diff
@@ -53,7 +55,7 @@ export const getJoinEmailTemplate = (siteUrl, sender, userEmail, team, isGuest =
         'Mattermost is a flexible, open source messaging platform that enables secure team collaboration.',
         'Learn more ( mattermost.com )',
         '',
-        '© 2021 Mattermost, Inc. 530 Lytton Avenue, Second floor, Palo Alto, CA, 94301',
+        `© ${currentYear} Mattermost, Inc. 530 Lytton Avenue, Second floor, Palo Alto, CA, 94301`,
```

Comment:
Maybe a better approach would be a map of year and server versions instead of current year since this will potentially fail when running tests against different server versions?

Reply:
https://github.com/mattermost/mattermost-mobile/pull/6059#discussion_r829610991